docs/UserGuide.md

```diff
@@ -92,6 +92,141 @@ Some example commands you can try:
 `DIRECTION` refers to the direction the player wants the shot to follow as displayed on the graph.
 Format: `shoot 2`
 
+### Shoot Command: `shoot`
+
+#### Overview
+
+The `shoot` command is an integral part of the game's penalty shootout mechanics, allowing the player to attempt to shoot a penalty by choosing one of the available directions. This command pits the player's choice against the AI's shot direction, determining whether the penalty is scored or not.
+
+#### Usage
+
+When it's the AI's turn to save, you will be prompted to select a direction to shoot. Enter the shoot command followed by your chosen direction. For example:
+
+shoot[direction] - Choose a direction to take a shot. Directions are represented by numbers (e.g., 0 for left, 1 for center, and 2 for right --> for beginners).
+
+#### Understanding Player Skill Levels
```

Comment:
I think this part about understanding player skill levels, is a bit repetitive to the section introducing Player
<img width="746" alt="image" src="https://github.com/AY2324S2-CS2113-F15-3/tp/assets/86831302/45b564df-75a9-4a36-b07a-6c293600828e">
Maybe you can remove this part.